documentation/projects/proposals/publish_dataset/20230706-project_proposal.md

```diff
@@ -0,0 +1,225 @@
+
+- [x] @AetherUnbound
+- [x] @sarayourfriend
+
+## Project summary
+
+This project aims to publish and regularly update a dataset or _datasets_ of
+Openverse's media metadata. This project will provide access to the data
+currently served by our API, but which is difficult to access in full and
+requires significant time, money, and compute resources to maintain.
+
+## Motivation
+
+For this project, understanding "why?" we would do this is of paramount
+importance. There are several key philosophical and technical advantages to
+sharing this dataset with the public.
+
+### Open Access + Scraping Prevention
+
+Openverse is and always has been, since its days as
+["CC Search"](https://creativecommons.org/2021/12/13/dear-users-of-cc-search-welcome-to-openverse/),
+informed by principles of the open access movement. Openverse strives to remove
+all all financial, technical, and legal barriers to accessing the works of the
+global commons.
+
+Due to technical and logistical limitations, we have previously been unable to
+accessibly provide access to the full Openverse dataset. Today, users need to
+invest significant time and money into scraping the Openverse API in order to
+access this data. These financial and technical barriers to our users are deeply
+inequitable. Additionally, this scraping disrupts Openverse access and stability
+for all users. It also requires significant maintainer effort to identify,
+mitigate, and block scraping traffic.
+
+By sharing this data as a full dataset on
+[HuggingFace](https://huggingface.co/datasets), we can remove these barriers and
+allow folks to access the data provided by Openverse.org and the Openverse API
+without restriction.
+
+### Contributions Back to Openverse
+
+Easily accessed Openverse datasets will facilitate easier generation of machine
+labels, translations, and other supplemental data which can be used to improve
+the experience of Openverse.org and the API. This data is typically generated as
+part of the
+[data preprocessing](https://huggingface.co/docs/transformers/preprocessing)
+stage of model training.
+
+Presence on HuggingFace in particular will enable community members to analyze
+the dataset and create supplemental datasets; to train models with the dataset;
+and to use the dataset with all of HuggingFace's tooling: the
+[Datasets](https://github.com/huggingface/datasets) library in particular.
+
+It is worth noting that this year we identified many projects to work on which
+rely on bulk analysis of Openverse's data. These projects could be replaced by,
+or made easier by the publication of the datasets. This could work in a few
+ways. A community member, training a model using the Openverse dataset,
+generates metadata that we want and planned to generate ourselves. Then, the
+HuggingFace platform presents an alternative to other SaSS products we intended
```

Comment:
Is "SaSS" the short version of something? I can't find a different meaning other than the CSS extension language, [SASS](https://sass-lang.com/). Can we add the full form or the meaning in a note/footnote, maybe?

Reply:
typo for Saas (software as a service), I'll update to explain.